platform/ios/docs/guides/Using Style Functions at Runtime.md

```diff
@@ -111,7 +111,7 @@ let stops: [Float: UIColor] = [
     10: .white,
 ]
 
-layer.circleColor = NSExpression(format: "FUNCTION(mag, 'mgl_stepWithMinimum:stops:', %@, %@)",
+layer.circleColor = NSExpression(format: "mgl_step:from:stops:(mag, %@, %@)",
                                  UIColor.green, stops)
 ```
 
```

Comment:
Right below here is an example that could potentially use `MGL_MATCH`. On the other hand, I do find the use of a dictionary with `valueForKeyPath:` to be somewhat elegant, so perhaps all it needs is a call to `mgl_coalesce:` to avoid the redundant `valueForKeyPath:` call.